datalad/downloaders/tests/test_http.py

```diff
@@ -199,6 +203,18 @@ def test_authenticate_external_portals():
 test_authenticate_external_portals.tags = ['external-portal', 'network']
 
 
+@skip_if_no_network
+def test_download_ftp():
+    try:
```

Comment:
codecov reports no coverage on this test, so I suspect that it is the same/related what we observed with @bpoldrack before -- may be there is no merge of coverage reports from multiple runs... asked on codecov chat and will report back